site/components/Navigation/index.tsx

```diff
@@ -332,57 +314,21 @@ export const Navigation: FC<Props> = ({
             />
             <NavItemMobile
               name={t({ message: "Infinity", id: "shared.infinity" })}
-              toggledNavItemID={toggledNavItemID}
-              setToggledNavItemID={setToggledNavItemID}
               url="/infinity"
               id="Infinity"
-              // subMenu={
-              //   [
-              // <NavItemMobile
-              //   name={t({
-              //     message: "INTRODUCTION",
-              //     id: "shared.intoduction",
-              //   })}
-              //   key="intro"
-              //   url="/infinity"
-              //   active={activePage === "Infinity"}
-              // />,
-              // <NavItemMobile
-              //   name={t({
-              //     message: "HOW TO PLEDGE",
-              //     id: "shared.how_to_pledge",
-              //   })}
-              //   key="how pledge"
-              //   // not sure about this url
-              //   url="/blog"
-              // />,
-              // <NavItemMobile
-              //   name={t({
-              //     message: "LEADER BOARD",
-              //     id: "shared.leader_board",
-              //   })}
-              //   key="leader board"
-              //   url="/blog"
-              // />,
-              //   ]
-              // }
             />
             <NavItemMobile
               name={t({ message: "Love Letters", id: "shared.loveletters" })}
-              toggledNavItemID={toggledNavItemID}
-              setToggledNavItemID={setToggledNavItemID}
               id="Love Letters"
               url={urls.loveletter}
             />
             <NavItemMobile
               name={t({ message: "Resources", id: "shared.resources" })}
-              toggledNavItemID={toggledNavItemID}
-              setToggledNavItemID={setToggledNavItemID}
               id="Resources"
```

Comment:
Delete `id` ... and so on

Reply:
All, ids deleted. I'm not sure its showing that they are there bc they were deleted in the commit a few days ago. 🤷 they are deleted tho haha.